libopenage/util/profiler.h

```diff
@@ -113,9 +113,9 @@ class Profiler {
 private:
 	void draw_canvas();
 	void draw_legend();
-	void draw_component_performance(std::string com);
+	void draw_component_performance(const std::string& com);
 	double duration_to_percentage(std::chrono::high_resolution_clock::duration duration);
-	void append_to_history(std::string com, double percentage);
+	void append_to_history(const std::string& com, double percentage);
```

Comment:
all the new `const &` should have their `&` on the right side attached (here to `&com`).